addons/account/tests/test_account_move_reconcile.py

```diff
@@ -2655,3 +2655,81 @@ def test_caba_foreign_vat(self):
         caba_move = self.env['account.move'].search([('tax_cash_basis_origin_move_id', '=', invoice.id)])
 
         self.assertEqual(caba_move.fiscal_position_id, foreign_vat_fpos, "The foreign VAT fiscal position should be kept in the the cash basis move.")
+
+    def test_caba_tax_group(self):
+        """ Test the CABA entries generated from an invoice with
+        a tax group
+        """
+        # Make the tax account reconcilable
+        self.tax_account_1.reconcile = True
+
+        # Create an invoice with a CABA tax using 'Include in analytic cost'
+        move_form = Form(self.env['account.move'].with_context(default_move_type='in_invoice', account_predictive_bills_disable_prediction=True))
+        move_form.invoice_date = fields.Date.from_string('2019-01-01')
+        move_form.partner_id = self.partner_a
+
+        tax_a = self.cash_basis_tax_a_third_amount
+        tax_b = self.cash_basis_tax_tiny_amount
+
+        tax_group = self.env['account.tax'].create({
+            'name': 'tax group',
+            'amount_type': 'group',
+            'company_id': self.company_data['company'].id,
+            'children_tax_ids': [(6, 0, [tax_a.id, tax_b.id])],
+        })
+
+        # line with analytic account, will generate 2 lines in CABA move
+        invoice = self.env['account.move'].create({
+            'partner_id': self.partner_a.id,
+            'invoice_date': fields.Date.from_string('2019-01-01'),
+            'move_type': 'entry',
+            'line_ids': [
+                # Base Tax line
+                (0, 0, {
+                    'debit': 0.0,
+                    'credit': 3000.0,
+                    'amount_currency': 3000.0,
+                    'account_id': self.company_data['default_account_revenue'].id,
+                    'tax_ids': [(6, 0, tax_group.ids)],
```

Comment:
Same here with `Command.set()` :wink:

Reply:
I forgot that we have this feature in 15
It's more readable now, thanks :D